src/pool/srv_pool.c

```diff
@@ -2199,6 +2189,67 @@ ds_pool_query_handler(crt_rpc_t *rpc)
 		D_GOTO(out_map_version, rc);
 	out->pqo_prop = prop;
 
+	if (DAOS_FAIL_CHECK(DAOS_FORCE_PROP_VERIFY)) {
```

Comment:
hmm, probably export the prop to the user, then you can query the prop by pool_query, and verify these information on client side.

Reply:
now daos_pool_query can query the prop already.
pool_iv_prop_fetch is only exported for server to fetch it through IV, as there is no usage now so here just test its correctness.

Reply:
sorry, I may miss sth. if the daos_pool_query can query the prop, why can not we fetch on client side? or I miss sth?

Reply:
client-side fetch it by sending RPC -> server serve the RPC by reading from RDB and return to client.
at server-side it caches it in IV.